src/Components/Aspire.Microsoft.EntityFrameworkCore.SqlServer/AspireSqlServerEFCoreSqlClientExtensions.cs

```diff
@@ -113,6 +113,42 @@ void ConfigureRetry()
 #pragma warning disable EF1001 // Internal EF Core API usage.
             if (!settings.DisableRetry || settings.CommandTimeout.HasValue)
             {
+#if NET9_0_OR_GREATER
+                builder.CheckDbContextRegistered<TContext>();
```

Comment:
Why is this only inside the if NET9 check? In the other EF libraries, it is for both.